whisperv6/utils.go

```diff
@@ -0,0 +1,136 @@
+package whisperv6
```

Comment:
I would extract mailserver related functions to `mailserver_response.go`.

Reply:
I can just rename the file, it is all about mailserver :)